app/services/user_service.js

```diff
@@ -211,10 +211,23 @@ module.exports = {
    */
   configureNewOtpKey: function (externalId, code, secondFactor, correlationId) {
     if (!externalId) {
-      Promise.reject('No externalId specified')
+      Promise.reject(new Error('No externalId specified'))
     }
 
-    return getAdminUsersClient({correlationId: correlationId}).configureNewOtpKey(externalId, code, secondFactor)
-  }
+    return getAdminUsersClient({ correlationId: correlationId }).configureNewOtpKey(externalId, code, secondFactor)
+  },
 
+  /**
+   * @param newPhoneNumber
+   * @returns {Promise}
+   */
+  updatePhoneNumber: (externalId, newPhoneNumber) => {
+    return new Promise((resolve, reject) => {
+      getAdminUsersClient().updatePhoneNumberForUser(externalId, newPhoneNumber)
+        .then(
+          () => resolve(),
+          error => reject(new Error(`There has been a problem updating the phone number: ${error}`))
```

Comment:
I think favour `.catch` instead of `(success(), failure())` it reads much more clearly, something like: 

```
getAdminUsersClient().updatePhoneNumberForUser()
  .then(() => {})
  .catch((error) => {})
```

Reply:
EDIT: 

This is using a `Promise` for the request anyway, it would be good to avoid any new Promises as it's much easier to make mistakes, I think this can be simplified with: 

```
return 
  getAdminUsersClient().updatePhoneNumberForUser(externalId, newPhoneNumber)
  .catch((error) => new Error(`There has been a problem updating the phone number: ${error}`))
```

Reply:
Yeah nice, I had just copy/pasted something else but this is better